portfolio/src/main/webapp/index.html

```diff
@@ -44,85 +53,100 @@ <h2>Background</h2>
-            <p class="overlay">Snapchat Political Ads Spending vs Number of Views - Data Science Project</p>
+      <div id="proj-anchor"></div>
+      <div id="proj-container">
+        <h2>Projects:</h2>
+        <div id="projects">
+          <a href="https://drive.google.com/file/d/1Jb52Jk0HLoNbdVDg3z5VrQMrCsxFrkv-/view?usp=sharing" target="_blank" onmouseover="revealOnMouseOver(this);" onmouseout="hideOnMouseOut(this);">
+            <div class="proj-container">
+              <img class="proj-img background" src="images/snapchat.jpg">
+              <p class="overlay visible">Snapchat Political Ads Spending vs Number of Views - Data Science Project</p>
+              <p class="overlay hidden">Given the dataset for Snapchat Poltical Ads, we had to clean the data, perform EDA, 
+                  assess missingness in the data, and finally perform a hypothesis test which determines whether there is a 
+                  difference in the amount of ads targeting Spanishing Speakers between the most expensive ads and the rest 
+                  of the ads.
+              </p>
+            </div>
+          </a>
+          <a href="https://amuamushu.github.io/dsc106-bts/" target="_blank" onmouseover="revealOnMouseOver(this);" onmouseout="hideOnMouseOut(this);">
+            <div class="proj-container">
+              <div id="gradient-bts" class="background"></div>
+              <p class="overlay visible">BTS's Rise in Popularity - Data Visualization Project</p>
+              <p class="overlay hidden"> As the final project for my Introduction to Data Visualization course, I created 
+                  data visualizations using HTML/CSS, Javascript, and the Highcharts API to tell the story of how BTS 
+                  rose in popularity over the years. The data was collected from various sources such as Google Trends, 
+                  Spotify, and Youtube Charts.
+              </p>
+            </div>
+          </a>
+          <a href="https://github.com/amuamushu/cogs9_final_project" target="_blank" onmouseover="revealOnMouseOver(this);" onmouseout="hideOnMouseOut(this);">
+            <div class="proj-container">
+                <!-- TODO: double check class id order -->
+              <img class="proj-img background" src="images/emoji.png" id="overlay-uni">
+              <div id="gradient-uni" class="background"></div>
+              <p class="overlay visible">University Sentiment vs Admissions Rate - Sentiment Analysis and Webscraping Class Project</p>
+              <p class="overlay hidden">This project was done for COGS 9: Introduction to Data Science and aims to answer a question 
+                  revolving UCSD's enrollment rate: whether there was correlation betweeen media influence, as determined by 
+                  positive/negative sentiment from news articles on UCSD and the school's US World News ranking, and the number 
+                  of students enrolling every year.
```

Comment:
This sentence should be split into multiple sentences.

Reply:
Done!